idbmms-connector/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencyManagement {
 
 dependencies {
   implementation project(':grpc-common')
+  implementation project(':auth-connector')
```

Comment:
Why is this required here? Client should not rely on the auth-connector.
Maybe you wanted auth common if you use some Crn magic?

Reply:
This is needed for `String requestId = RequestIdUtil.getOrGenerate(MDCUtils.getRequestId());`

Reply:
You could use this version of to generate a RequestId:
https://github.com/hortonworks/cloudbreak/blob/1fbceb9382a26df2277e1d6f2df2839f46f48e6f/common/src/main/java/com/sequenceiq/cloudbreak/logger/MDCBuilder.java#L69-L69